http/jsonapi/runtime/error.go

```diff
@@ -118,6 +116,7 @@ func WriteError(w http.ResponseWriter, code int, err error) {
 		if source := ei.Source; source != nil {
 			ev = ev.Fields(*source)
 		}
+		ev.Msg("")
```

Comment:
Had a linter issue that ev is unused. Is this what you intended here @threez

Reply:
Hmm, yes the event is build based on the parameters and then issued with `.Msg(...)`. Why is the linter complaining?

Reply:
I added the `ev.Msg`. This code was not there before that's why the linter complained that `ev` was unused.

Reply:
We should delete the loop to avoid multiple info lines or only log on debug level. The msg should not be empty, but contain at least `.Msg("Send error to client")`

Reply:
done

Reply:
Please move discussion to #231